src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -31,6 +31,8 @@ namespace System.Collections
 #if FEATURE_NETCORE
     [FriendAccessAllowed]
 #endif
+    /// <summary>Implements the <see cref="T:System.Collections.IList" /> interface using an array whose size is dynamically increased as required.To browse the .NET Framework source code for this type, see the Reference Source.</summary>
```

Comment:
> @Priya91, can you comment here? We use the Roslyn code formatter, so if we can configure it to do this, then yes.

@cartermp I used the Roslyn Syntax Visualizer tool on Visual studio. It shows the whole 
```
/// <summary>
/// aaaaaaaaaaaaaaaaa
/// bbbbbbbbbbbbbbbbb
/// </summary>
```
as a ```SingleLineDocumentationCommentTrivia```, and the ``` /// aaaaaaaaaaa ``` part as ```XmlTest``` with the following child nodes, ```XmlTextLiteralToken``` and ```XmlTextLiteralNewLineToken```

So you would have to create a ```XmlText``` node with ```XmlTextLiteralNewLineToken``` inserted after each ```XmlTextLiteralToken```, and the number of ```XmlTextLiteralToken``` will be equal to the number of n char blocks, where n is the length of line in chars. You also need to figure out a way to not cut words, that is split the string at whitespace chars.

I don't think there is a single Roslyn API that will do this for you. @srivatsn Is there?